packages/components/src/styles/components/page-header.scss

```diff
@@ -79,6 +78,10 @@
   display: flex;
   flex-direction: column;
   gap: 4px;
+
+  > *:first-child {
```

Comment:
It may depend on how specific the selector they're using is, but if they just used `.hds-page-header__subtitle` for example then I think this would override that.